packages/app/src/lib/audit.ts

```diff
@@ -51,6 +51,16 @@ async function hasPasswordBeenCompromised(passwordHash: string) {
     return false;
 }
 
+async function isItemExpiringOrExpired(expiryDate: Date) {
+    const thirtyDaysFromNow = add(new Date(), { days: 30 });
```

Comment:
I assume this is for testing?

Reply:
Not really, I thought it made sense to warn in the UI when an item is expiring too, so people know it needs to be acted on, rather than waiting for it to be "past" its date.